presentation.js

```diff
@@ -150,7 +177,7 @@ define(function(require, exports, module) {
             togglePresentationMode(false);
             menuItem = null;
             barExtras = null;
-            presenting = false;
+            presenting = undefined;
```

Comment:
Probably more apt to do `delete presenting;`!

Reply:
I don't think `delete` can delete variables?

Reply:
Ah, TIL, seems you only can if it's not declared with `var`.